### PR TITLE
修复实时音视频相关的bug；更新群组；添加聊天室列表页面

### PR DESCRIPTION
**Pull request by xieyajie**
fix: 1.iOS bug: iPhone is not ringing when making a call nor receiving a call.

2. iOS bug: No push notification if the app is terminated.

### CODE REVIEW

File: ChatDemo-UI3.0/ChatDemo-UI3.0/Code/Controllers/Call/EaseCallViewController.m [hunk 1]
```diff
@@ -330,6 +347,28 @@ - (void)close
...
+- (void)_beginRing
+{
+    NSString *musicPath = [[NSBundle mainBundle] pathForResource:@"callRing" ofType:@"mp3"];
+    NSURL *url = [[NSURL alloc] initFileURLWithPath:musicPath];
+    
+    self.ringPlayer = [[AVAudioPlayer alloc] initWithContentsOfURL:url error:nil];
+    [self.ringPlayer setVolume:1];
+    self.ringPlayer.numberOfLoops = -1; //设置音乐播放次数  -1为一直循环
+    if([self.ringPlayer prepareToPlay]) {
+        [self.ringPlayer play]; //播放
```
Comment (githubzhaoliang) [inline ChatDemo-UI3.0/ChatDemo-UI3.0/Code/Controllers/Call/EaseCallViewController.m]: remove above chinese comments

Comment (xieyajie) [reply ChatDemo-UI3.0/ChatDemo-UI3.0/Code/Controllers/Call/EaseCallViewController.m]: done

File: ChatDemo-UI3.0/ChatDemo-UI3.0/Code/Controllers/Groups/Info/EMAddAdminViewController.h [hunk 2]
```diff
@@ -0,0 +1,15 @@
+//
+//  EMAddAdminViewController.h
+//  ChatDemo-UI3.0
+//
+//  Created by XieYajie on 16/03/2017.
+//  Copyright © 2017 easemob. All rights reserved.
```
Comment (jerrywu12) [inline ChatDemo-UI3.0/ChatDemo-UI3.0/Code/Controllers/Groups/Info/EMAddAdminViewController.h]: need to use hyphenate instead of easemob.

**Comment by jerrywu12** [comment]
has warning during pod update, "[!] Automatically assigning platform ios with version 10.3 on target ChatDemo-UI3.0 because no platform was specified. Please specify a platform for this target in your Podfile. See `https://guides.cocoapods.org/syntax/podfile.html#platform`."

**Comment by jerrywu12** [comment]
@xieyajie Is there chat room added/removed delegate for EMChatRoomViewController.m? 
Seem like it only load the chat room object by calling the method getChatroomsFromServerWithPage:pageSize:completion:
like the notification for EMGroupViewController.m, 
#define KEM_REFRESH_GROUPLIST_NOTIFICATION     @"refreshGroups_notification"

**Comment by jerrywu12** [comment]
it seems ready to merge beside 2 issues mentioned above.

File: ChatDemo-UI3.0/ChatDemo-UI3.0/Code/Controllers/Groups/Info/EMAddAdminViewController.h (hunk 2 above)
Comment (xieyajie) [reply ChatDemo-UI3.0/ChatDemo-UI3.0/Code/Controllers/Groups/Info/EMAddAdminViewController.h]: done

**Comment by xieyajie** [comment]
@jerrywu12 In the EMChatroomsViewController has added KEM_DESTROY_CHATROOM_NOTIFICATION. After destroy a chatroom,  EMChatroomsViewController will update datasource and reload tableview.

**Pull request merged**